index.html

```diff
@@ -511,14 +511,15 @@
   // The Present button is visible if at least one presentation display is available
   var presentBtn = document.getElementById("presentBtn");
   // It is also possible to use relative presentation URL e.g. "presentation.html"
-  var presUrl = "http://example.com/presentation.html";
+  var presUrls = ["http://example.com/presentation.html",
+                  "http://another.com/alternate.html"];
```

Comment:
Examples should all use `example.com`, `example.net`, `example.org` or subdomains:
http://www.w3.org/2001/06/manual/#Examples

I suggest to switch to: `http://example.net/alternate.html`